tests/files/dbc/string_signals.dbc

```diff
@@ -0,0 +1,41 @@
+VERSION ""
+
+
+NS_ : 
+	NS_DESC_
+	CM_
+	BA_DEF_
+	BA_
+	VAL_
+	CAT_DEF_
+	CAT_
+	FILTER
+	BA_DEF_DEF_
+	EV_DATA_
+	ENVVAR_DATA_
+	SGTYPE_
+	SGTYPE_VAL_
+	BA_DEF_SGTYPE_
+	BA_SGTYPE_
+	SIG_TYPE_REF_
+	VAL_TABLE_
+	SIG_GROUP_
+	SIG_VALTYPE_
+	SIGTYPE_VALTYPE_
+	BO_TX_BU_
+	BA_DEF_REL_
+	BA_REL_
+	BA_DEF_DEF_REL_
+	BU_SG_REL_
+	BU_EV_REL_
+	BU_BO_REL_
+	SG_MUL_VAL_
+
+BS_:
+
+BU_:
+
+
+BO_ 10 PersonName: 40 Vector___XXX
+ SG_ first_name : 0|160@1+ (1,0) [0|0] "String" Vector__XXX
+ SG_ last_name : 160|160@1+ (1,0) [0|0] "str" Vector__XXX
```

Comment:
You might want to have a test case that includes some other types in it as well to make sure there is no offset issues.  I'd suggest a test that has all types (or if you already have that, add strings to that test case).